src/CMakeLists.txt

```diff
@@ -637,9 +637,14 @@ install(TARGETS ceph-mon DESTINATION bin)
 if(NOT ALLOCATOR STREQUAL "jemalloc")
   set(disable_jemalloc "DISABLE_JEMALLOC=1")
 endif()
+set(EXTRA_CXXFLAGS -fPIC -Wno-unused-variable -DNDEBUG)
+if(FREEBSD)
+  set(EXTRA_CXXFLAGS ${EXTRA_CXXFLAGS} -Irocksdb/include -I/usr/local/include)
+  set(MAKEFLAGS ${MAKEFLAGS})
```

Comment:
@tchaikov 
It is possible to run the buidl with `make V=1 VERBOSE=1` to get e more verbose output of what is going on.
And the build in RocksDB supports that as well.